inspirehep/base/templates/format/record/Inspire_Default_HTML_detailed.tpl

```diff
@@ -17,60 +17,93 @@
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 #}
 
-{% from "format/record/Inspire_HTML_detailed_macros.tpl" import record_buttons, record_collection_heading, record_title, record_collections, record_publication_info, record_doi, record_links, detailed_record_abstract, record_keywords, record_references, record_citations, record_plots with context %}
 
-{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import render_record_authors, record_cite_modal, record_arxiv with context %}
+{% from "format/record/Inspire_HTML_detailed_macros.tpl" import record_buttons, record_collection_heading, record_collections, record_publication_info, record_doi, record_links, detailed_record_abstract, record_keywords, record_references, record_citations, record_plots with context %}
+
+{% from "format/record/Inspire_Default_HTML_general_macros.tpl" import mathjax, render_record_title, render_record_authors, record_cite_modal, record_arxiv, record_report_numbers with context %}
 
 <div class="record-detailed">
   <div class="record-header">
-    {{ record_cite_modal() }}
     <div id="hep-collection" class="record-collection-heading ellipsis">
       {{ record_collection_heading() }}
     </div>
-    <div id="record-title">
-      {{ record_title() }}
-    </div>
-    <div id="record-authors">
-      {{ render_record_authors(is_brief=false, number_of_displayed_authors=25) }}
-    </div>
-    <div class="journal">
-      {{ record_publication_info() }}
-    </div>
-    <div id="doi-eprint-experiment">
-      {{ record_doi() }}
-      {{ record_arxiv(is_brief=false) }}
-      <div class="cite-pdf-buttons">
-        <div class="btn-group">
-          {{ record_buttons() }}
+    <div class="row">
+        {{ record_cite_modal() }}
+      <div class="col-md-10" id="left-column">
```

Comment:
Do we need an id here or just a class? When naming `ids` we should add some prefix to their name depending on the context where they are. We have seen already some id name clashing.